rpcserver.go

```diff
@@ -2663,12 +2665,35 @@ func createRPCOpenChannel(r *rpcServer, graph *channeldb.ChannelGraph,
 	}
 	externalCommitFee := dbChannel.Capacity - sumOutputs
 
+	chanID := dbChannel.ShortChannelID.ToUint64()
+
+	// Get the lifespan observed by the channel event store. It it is unknown,
+	// zero values will be returned (which wil yield a zero lifetime).
```

Comment:
If the channel is unknown, shouldn't we then stop making additional calls?

Reply:
I think it will be handled correctly as it currently is?